tests/unit/phase/map.py

```diff
@@ -1,10 +1,12 @@
 #!/usr/bin/env -S pytest -svv
+import pytest
 
 from io import StringIO
 from digital_land.pipeline import run_pipeline
 from digital_land.phase.load import LoadPhase
 from digital_land.phase.parse import ParsePhase
 from digital_land.phase.map import MapPhase
+from digital_land.phase.map import normalise as map_normalise
```

Comment:
why has this changed name? It doesn't seem necessary.

Reply:
I'll change it.